textile/chat-features.textile

```diff
@@ -409,6 +410,7 @@ Broadly speaking, messages are published via REST calls to the Chat HTTP API and
 * @(CHA-M6b)@ @[Testable]@ If the REST API returns an error, then the method must throw its @ErrorInfo@ representation.
 * @(CHA-M13)@ A single message must be retrievable from the REST API.
 ** @(CHA-M13a)@ @[Testable]@ A method must be exposed that accepts a message serial and returns a single @Message@ object. It shall call the "REST API single message endpoint"#rest-fetching-single-message and return the message if found.
+*** @(CHA-M13a1)@ @[Testable]@ If not found an error should be thrown with an error code 40400 and status code 404.
```

Comment:
Isn't this covered by  CHA-M13b (the REST API throws this combination as it stands)?

Reply:
Have asked about this in https://ably-real-time.slack.com/archives/CURL4U2FP/p1762431489639239